Assets/MixedRealityToolkit/Utilities/Editor/ScriptingUtilities.cs

```diff
@@ -60,6 +42,32 @@ public class ScriptingUtilities
 
             PlayerSettings.SetScriptingDefineSymbolsForGroup(targetGroup, defines);
         }
+
+        /// <summary>
+        /// Removes a set of symbolic constant definitions to Unity's Scripting Define Symbols from the
+        /// specified build target group.
+        /// </summary>
+        /// <param name="targetGroup">The build target group for which the sybmols are to be removed.</param>
+        /// <param name="symbols">Array of symbols to remove.</param>
+        public static void RemoveScriptingDefinitions(
+            BuildTargetGroup targetGroup,
+            string[] symbols)
+        {
+            if (symbols == null || symbols.Length == 0) { return; }
+            List<string> toRemove = new List<string>(symbols);
+
+            string[] oldDefines = PlayerSettings.GetScriptingDefineSymbolsForGroup(targetGroup).Split(';');
+            List<string> defines = new List<string>();
+
+            foreach (string s in oldDefines)
```

Comment:
> foreach (string s in oldDefines) [](start = 11, length = 33)

Normally I wouldn't mention this because it involves LINQ, but because this is editor/scripting code (which also appears to be roughly run-once  or at least once-per-import of stuff):

[https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.except?view=netframework-4.8](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.except?view=netframework-4.8)

Reply:
updated